test/integration/resources/shoot-gcp.yaml

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
Why is this file here? We have templates in `hack/templates/resources` and examples, can we please reuse them?

Reply:
because I stripped off the name and the domain from the resource example. The test generates those for you. That said, I could still remove it, and add a mention in the documentation.